src/components/Amount.tsx

```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
 
```

Comment:
Why is cKES behind eXOF although it has more outstanding supply? everything else is ordered by supply

<img width="1086" alt="image" src="https://github.com/user-attachments/assets/0e8099e3-2e75-4134-be5a-2ee4d444ac85">

Reply:
It's just a coincidence. There's no sorting. I've added sorting, it wasn't specified previously or noted by Roman or Ole in any of the past several reviews

Reply:
They're in release order